README.md

```diff
@@ -44,7 +44,7 @@ PINTS comes with a number of [detailed examples](examples/README.md), hosted her
 In addition, there is a [full API documentation](http://pints.readthedocs.io/en/latest/), hosted on readthedocs.io.
 
 
-## Installing PINTS
+## Installing PINTS 
```

Comment:
Is this a crucial space? Because my editor will cut this out next time I open the file :D

Reply:
That was quick! No it's not, I just undid a change I made and didn't notice the space :-|

Reply:
Fixed.